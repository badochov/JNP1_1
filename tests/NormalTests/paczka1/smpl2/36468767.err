Error in line 11: XQwvBhlT S799 -6,7
Error in line 8: NBgFSPs S49 74,1
Error in line 13: NBgFSPs S111 54,2
Error in line 7: XQwvBhlT S799 88,8
Error in line 25: HZkV8W8R S799 86,6
Error in line 10: OQVs7 S799 90,7
Error in line 26: HZkV8W8R S49 8,0
