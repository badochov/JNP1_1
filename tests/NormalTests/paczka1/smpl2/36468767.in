OQVs7 S799 65,2
OQVs7 S799 71,7
HZkV8W8R S49 40,2
A736 A736 89,8
A736 A736 115,8
HZkV8W8R S49 186,5
XQwvBhlT S799 88,8
NBgFSPs S49 74,1
j1mUg440 S111 18,6
OQVs7 S799 90,7
XQwvBhlT S799 -6,7
?
NBgFSPs S111 54,2
?S111
A736 S111 99,9
?
NBgFSPs S799 20,6
NBgFSPs S799 55,3
A736 S111 12,0
j1mUg440 S111 88,0
WijYoMW A736 60,6
XQwvBhlT S111 22,0
XQwvBhlT S111 35,6
NBgFSPs S49 80,8
HZkV8W8R S799 86,6
HZkV8W8R S49 8,0
?S799
?S799
?A736
OQVs7 S49 46,9
WijYoMW A736 117,0
NBgFSPs S49 104,2
OQVs7 S49 33,3
HZkV8W8R A736 0,5
?j1mUg440
WijYoMW S799 22,2
XQwvBhlT S799 45,0
OQVs7 A736 16,6
NBgFSPs S799 44,5
HZkV8W8R A736 1,4