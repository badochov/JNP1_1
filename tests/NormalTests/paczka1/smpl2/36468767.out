A736 A 26,0
HZkV8W8R S 146,3
OQVs7 S 6,5
S49 146,3
A736 26,0
S799 6,5
A736 A 26,0
HZkV8W8R S 146,3
OQVs7 S 6,5
S49 146,3
A736 26,0
S799 6,5
S799 41,2
S799 41,2
A736 A 26,0 S 87,9
A736 26,0
j1mUg440 S 69,4
